app/src/main/java/com/github/multimatum_team/multimatum/activity/MainActivity.kt

```diff
@@ -18,8 +18,12 @@ import androidx.core.content.ContextCompat
 import com.github.multimatum_team.multimatum.R
 import com.github.multimatum_team.multimatum.adaptater.DeadlineAdapter
 import com.github.multimatum_team.multimatum.repository.DeadlineRepository
+import com.github.multimatum_team.multimatum.repository.GroupRepository
 import com.github.multimatum_team.multimatum.util.DeadlineNotification
 import com.github.multimatum_team.multimatum.viewmodel.DeadlineListViewModel
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseAuthSettings
+import com.google.firebase.firestore.FirebaseFirestore
```

Comment:
I only see in this activity that you have removed two empty line. Are these imports still useful ?

Reply:
Good catch